codesystems/CodeSystem-England-HTTPErrorCodes.xml

```diff
@@ -0,0 +1,263 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+  <id value="England-HTTPErrorCodes" />
```

Comment:
you stated that these have been changed to singular? e.g. `England-HTTPErrorCode`?